<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LinuxProjectModifierClass</class>
 <widget class="QMainWindow" name="LinuxProjectModifierClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LinuxProjectModifier</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonSelectFolder">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>141</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Folder</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>691</width>
      <height>221</height>
     </rect>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <column>
     <property name="text">
      <string>Path</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Project</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButtonExecute">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>380</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>実行</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>541</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;サブフォルダの検索&lt;/p&gt;&lt;p&gt;proファイルを検索&lt;/p&gt;&lt;p&gt;プロジェクト内AdditionalIncludeDirectoriesの取り出し&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSelectQtLib">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Qt Lib Folder</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSelectQtInclude">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>171</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Qt Include Folder</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditQtLib">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>13</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditQtInclude">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
